data/recipes/gcp_forensics.json

```diff
@@ -11,22 +11,25 @@
           "remote_instance_name": "@instance",
           "incident_id": "@incident_id",
           "zone": "@zone",
+          "create_analysis_vm": "@analysis_vm",
           "disk_names": "@disks",
           "all_disks": "@all_disks",
           "boot_disk_size": "@boot_disk_size",
           "boot_disk_type": "@boot_disk_type",
-          "cpu_cores": 16,
+          "cpu_cores": "@cpu_cores",
           "image_project": "ubuntu-os-cloud",
           "image_family": "ubuntu-1804-lts"
       }
   }],
   "args": [
       ["remote_project_name", "Name of the project containing the instance / disks to copy ", null],
-      ["incident_id", "Incident ID to label the VM with.", null],
+      ["analysis_project_name", "Name of the project where the analysis VM will be created", null],
+      ["--incident_id", "Incident ID to label the VM with.", null],
```

Comment:
For the "incident-id" I made it optional since I am making the creation of the analysis VM optional, this was a ToDo comment that I ran through in the code and I think it make the recipe more flexible. It will not affect the existing way of invoking this recipe, it will still work. What do you think?

Reply:
I think in the interest of the user, I'd try to make the interface as consistent as possible across the different `gcp_` recipes.

Another point - you only create the `_gcp_label` attribute if an analysis VM is being created, but this label is also applied to disks. In the current logic it will only be applied to disks if the VM is created, which seems to be two different things. It might be better to adjust the code to apply a label based on `incident_id` no matter what

Reply:
both GCP recipes now have the --incident_id attribute as optional, so now its consistent. 
The disks will also get _gcp_label added however this will happen in the process method, while the created VM will have the _gcp_label in the startup method, but both are covered